ci/skeleton/etc/apache2/mods-available/hhvm_proxy_fcgi.conf

```diff
@@ -0,0 +1 @@
+ProxyPassMatch ^/(.+\.(hh|php|hack)(/.*)?)$ fcgi://127.0.0.1:9000/var/www/$1
```

Comment:
This PR effectively inlines the `packaging` repository in this directory instead of checking it out. Discussing this with Fred, this was discussed in the past and this approach was slightly superior as it ensures the CI and the code are kept in sync.